model-optimizer/extensions/front/onnx/pooling_ext.py

```diff
@@ -92,9 +92,8 @@ def common_onnx_pool_extractor(node):
     strides = onnx_attr(node, 'strides', 'ints', default=None, dst_type=lambda x: np.array(x, dtype=np.int64))
     final_strides = np.array([1, 1, *[x for x in strides]], dtype=np.int64) if strides is not None else None
 
-    dilations = onnx_attr(node, 'dilations', 'ints', default=None, dst_type=lambda x: np.array(x, dtype=np.int64))
-    assert dilations is None or np.all(dilations == 1),\
-        'Node {} has "dilations" attribute with values not equal to 1s which is not supported'.format(node.id)
+    dilation = onnx_attr(node, 'dilations', 'ints', default=None, dst_type=lambda x: np.array(x, dtype=np.int64))
+    final_dilation = np.array([1, 1, *[x for x in dilation]], dtype=np.int64) if dilation is not None else None
 
```

Comment:
is your onnx extract correct for all onnx versions of MaxPool? Please check attributes.

Reply:
If ONNX MaxPool has no a `dilations` attribute, we will set None here. This case will be handled in the infer function.